xcell/cls/cl.py

```diff
@@ -226,6 +226,23 @@ def get_cl_file(self):
                 cl -= nl
             # Note that while we have subtracted the noise
             # bias from `cl_cp`, `cl_cov_cp` still includes it.
+            if (mapper1.mask_power > 1) or (mapper2.mask_power > 1):
```

Comment:
We need a test for this if statement

Reply:
Done! I just re-added correction as an output of get_cl_file() so I can test whether it is different than 1 and if it matches the formula.